qr-code/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2021 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by josephj on 14/4/2021.
+  -->
+
+<resources>
+    <color name="progressTrackColor">#E6E9EB</color>
```

Comment:
Is this independent or should it be related to the theme? Like primaryColor or something?

Reply:
I didn't find it anywhere else so I guess it's only related to this screen.